File: Sargis.Muradyants/EmployeesTasks.Web/ClientApp/src/app/models/employee-task.ts
import { Employee } from './employee';
import { State } from './state';
import { PriorityLevel } from './priority-level';

export class EmployeeTask {

  public id: number;
  public title: string;
  public description: string;
  public estimate: string;
  public employee: Employee;
  public state: State;
  public prioritylevel: PriorityLevel;
}

File: Sargis.Muradyants/EmployeesTasks.Data/Entities/EmployeeTask.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace EmployeesTasks.Data.Entities
{
    public class EmployeeTask
    {
        public int Id { get; set; }        
        public string Title { get; set; }
        public string Description { get; set; }        
        public string Estimate { get; set; }
        public int EmployeeId { get; set; }
        public int PriorityLevelId { get; set; }
        public int StateId { get; set; }        
        public Employee Employee { get; set; }
        public PriorityLevel PriorityLevel { get; set; }
        public State State { get; set; }        
    }
}
